python_modules/dagster/dagster/core/instance/ref.py

```diff
@@ -50,12 +150,13 @@ class InstanceRef(
             ("run_storage_data", ConfigurableClassData),
             ("event_storage_data", ConfigurableClassData),
             ("compute_logs_data", ConfigurableClassData),
-            ("schedule_storage_data", Optional[ConfigurableClassData]),
+            ("schedule_storage_data", ConfigurableClassData),
```

Comment:
should we re-order these to the bottom under a comment about their required-but-legacy status?